types/vue-observe-visibility/tsconfig.json

```diff
@@ -0,0 +1,23 @@
+{
+    "compilerOptions": {
+        "module": "commonjs",
+        "lib": [
+            "esnext", "dom"
```

Comment:
Is this config required for this package to pass? It doesn't match the expected form.